.buildkite/scripts/pipelines/pull_request/pipeline.ts

```diff
@@ -329,9 +319,9 @@ const uploadPipeline = (pipelineContent: string | object) => {
     pipeline.push(getPipeline('.buildkite/pipelines/pull_request/post_build.yml'));
 
     // remove duplicated steps
-    uploadPipeline([...new Set(pipeline)].join('\n'));
```

Comment:
Does this need to be reverted? Are we able to run this PR against the new infrastructure (not KME)?

Reply:
The new infra is the same in this sense, it requires the pipeline to be emitted to STDOUT and not uploaded through the API.